test/features/update_httpport.feature

```diff
@@ -0,0 +1,56 @@
+Feature: Update the HTTP Port field in Kogito Services
```

Comment:
Do you think it makes sense to add so many scenarios for testing the update of HTTP port field feature? At the moment, we're running 1 for each kogito component (3 in total) and 3 more for quarkus, native and spring boot. Maybe we could check all the kogito components in the same scenario. What do you think about?

Reply:
I think it is ok to keep it separate, that way we can pinpoint which component is actually failing.